swiftlint.yml

```diff
@@ -0,0 +1,106 @@
+excluded:
```

Comment:
Why is this file added here?

Reply:
This is the rules files from [raywenderlich](https://github.com/raywenderlich/swift-style-guide/blob/main/com.raywenderlich.swiftlint.yml) 
You may suggest to add or remove anything

Reply:
I don't mind the rules themselves, as long as they're consistent. I would suggest reading [documentation on configuration](https://github.com/realm/SwiftLint#configuration). A configuration already exists and this file was not the driver for the SwiftLint you applied 😜

Reply:
@ehab-al-cko if you look at the checkout folder in this repo you will see a similar swiftlint.yml file, see here : https://github.com/checkout/frames-ios/blob/release/4.0.0_RC/Checkout/.swiftlint.yml. you can have the similar rules coming remotely from [raywenderlich](https://github.com/raywenderlich/swift-style-guide/blob/main/com.raywenderlich.swiftlint.yml). Just copy paste that file and also enable the bitrise checks to run these checks, which means you will have to change the bitrise yml file as well on this branch.